packages/manager/apps/key-management-service/src/pages/credential/CreateCredential.page.tsx

```diff
@@ -62,24 +79,39 @@ const CreateCredential = () => {
           headerButton: <KmsGuidesHeader />,
         }}
       >
-        <div className="w-full block">
-          <div className="mb-6">
-            <Notifications />
-          </div>
-          <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 md:gap-6">
-            <CreateGeneralInformations
-              name={name}
-              setName={setName}
-              validity={validity}
-              setValidity={setValidity}
-              description={description}
-              setDescription={setDescription}
-              csr={csr}
-              setCsr={setCsr}
-            />
+        <IdentityDataProvider>
+          <div className="w-full block">
+            <div className="mb-6">
+              <Notifications />
+            </div>
+            <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 md:gap-6">
```

Comment:
this div should be defined on each step to specify a corresponding layout. Right now this 3 column grid layout may fit the first step but does not on the second step.